CHANGELOG.md

```diff
@@ -10,6 +10,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 [Unreleased][Unreleased_log]
 --------------
 ### Added
+- `oe_set_host_log_level(oe_log_level_t log_level)` allows applications to dynamically modify the verbosity of logs captured from host.
+  - This API only modifies the verbosity of logs generated by the host. This does not affect enclave log level verbosity.
```

Comment:
We can remove this line to shorten the text. You can leave it in the design doc.